bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/metrics/BigtableClientMetrics.java

```diff
@@ -15,188 +15,30 @@
  */
 package com.google.cloud.bigtable.metrics;
 
-import com.codahale.metrics.Counting;
-import com.codahale.metrics.Metric;
-import com.codahale.metrics.MetricFilter;
-import com.codahale.metrics.MetricRegistry;
-import com.codahale.metrics.Reporter;
-import com.codahale.metrics.Slf4jReporter;
-
-import java.util.concurrent.TimeUnit;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 /**
  * Singleton Container for a {@link MetricRegistry}. The default behavior is to return
  * implementations that do nothing. Exporting of metrics can be turned on by either adding TRACE
  * level logging for this class, which will write out all metrics to a log file, or via a call to
- * {@link BigtableClientMetrics#enable} followed by a programmatic configuration of reporters as per
+ * {@link BigtableClientMetrics#setMetricRegistry(MetricRegistry)}, configuration of reporters as per
```

Comment:
"or via a call to {@link BigtableClientMetrics#setMetricRegistry(MetricRegistry)} with configuration of {@link Reporter}s as per"